recipes/strawberryperl/all/conanfile.py

```diff
@@ -40,4 +40,5 @@ def package_info(self):
         self.output.info("Appending PATH environment variable: %s" % bin_path)
         self.env_info.PATH.append(bin_path)
 
+        self.user_info.perl = os.path.join(self.package_folder, "bin", "perl.exe").replace("\\", "/")
         self.deps_user_info.perl = os.path.join(self.package_folder, "bin", "perl.exe").replace("\\", "/")
```

Comment:
I don't have any idea, if this line does anything, as OpenSSL uses `rootpath`. I kept it just in case.